UI/EncyclopediaDetailPanel.cpp

```diff
@@ -3009,8 +3009,7 @@ namespace {
                 float planet_capacity = ((planet_environment == PlanetEnvironment::PE_UNINHABITABLE) ?
                                          0.0f : planet->GetMeter(MeterType::METER_TARGET_POPULATION)->Current()); // want value after temporary meter update, so get current, not initial value of meter
 
-                retval.emplace(std::piecewise_construct, std::forward_as_tuple(planet_capacity),
-                               std::forward_as_tuple(std::move(species_name), planet_environment));
+                retval.emplace(planet_capacity, std::pair{species_name, planet_environment});
```

Comment:
leave the `piecewise_construct` and `forward_as_tuple`; just remove `std::move()`